packages/system/src/style.js

```diff
@@ -171,6 +171,21 @@ function indexGeneratorsByProp(styles) {
   return index
 }
 
+function getMediaOrder(styles, props) {
+  const medias = {} 
+  const breakpoints = Object.values(getBreakpoints(props))
```

Comment:
Please use `Object.keys` instead (ES5 & perf). Then iterate on it and get the breakpoint from `const breakpoint = breakpoints[key]`.